storagemarket/impl/provider.go

```diff
@@ -339,4 +361,8 @@ func (p *providerDealEnvironment) Disconnect(proposalCid cid.Cid) error {
 	return p.p.conns.Disconnect(proposalCid)
 }
 
+func (p *providerDealEnvironment) DealAcceptanceBuffer() abi.ChainEpoch {
```

Comment:
So I've gone back and forth a bunch on this, but I wonder if we should throw out the providerDealEnvironment and just make these methods on the Provider, cause we have the same aforementioned feature above -- the methods aren't accessible assuming you just have the storage provider interface. Non-blocking, but what do you think?

Reply:
I like implementing the interface on our provider impl; it would remove a bunch of indirection.